src/Microsoft.AspNet.Razor/CodeGenerators/CSharpCodeWriter.cs

```diff
@@ -470,23 +470,6 @@ private void WriteCStyleStringLiteral(string literal)
                         Write(literal[i].ToString());
                         break;
                 }
```

Comment:
If we want we could always render C# strings always as `@"hello world"` or `"\r\nhello\tworld"` but thought I'd reduce the amount of change by just nuking the below code that interferes with what the issue was geared towards. In the end this bit of code was from CodeDOM and I believe a lot of it was for readability.